imports/ui/RouterComponents/pages/EditPageContainer.js

```diff
@@ -14,7 +14,7 @@ export default EditPageContainer = withTracker((props) => {
       ? pages.find((page) => parseTitle(page.title) === pageId)
       : null;
 
-  const pageTitles = pages ? pages.map((page) => page.title) : [];
+  const pageTitles = pages ? pages.map((page) => page.title) : []
```

Comment:
what is the change here ? why semicolon deleted

Reply:
It was probably not a conscious change ;p

Reply:
Added the semicolon back.